INTEGRATE.md

```diff
@@ -0,0 +1,32 @@
+# Local integration between Porta/System with Keycloak through Zync
+
+## Install Porta
+Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
+
+## Install Zync
+Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
+
+## Install Keycloak
+You can have a keycloak running locally with `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
+So that docker container with keycloak will be accessible from your browser in `http://localhost:8080`.
+
+After that, you will need to follow the instructions to [configure Red Hat Single Sign-On](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/api_authentication/openid-connect).
+
+## Run Zync
+From the folder where you have Zync installed, run: `ZYNC_AUTHENTICATION_TOKEN=zynctoken bundle exec rails server -p 5000` so you will get zync running in the port 5000.
+
+## Run Porta
+If your porta is already running, stop it, and if you have `spring`, also do `spring stop`.
```

Comment:
does that make summer come early ?

Reply:
Then no need to do it, this is only for those who have Spring running. It is in the instructions to install Porta/System as optional:
https://github.com/3scale/porta/blob/master/INSTALL.md#spring-optional

> Spring is a Rails application preloader. It speeds up development by keeping your application running in the background so you don't need to boot it every time you run a test, rake task or migration.
> This is not required but still recommended. Install it via gem: `gem install spring -v 2.0.0`

Reply:
I think this is troubleshooting not really installation. And only Porta related. Is there any need to mention details of Porta in Zync?